python_liftbridge/python_liftbridge.py

```diff
@@ -72,7 +72,7 @@ def _create_stream(self, stream_request):
 
     @handle_rpc_errors
     def _publish(self, publish_request):
-        response = self.stub.Publish(publish_request)
+        response = self.stub.PublishToSubject(publish_request)
```

Comment:
Why is this changing to `PublishToSubject`? This function is for publishing to a stream, right?

Reply:
There should be a separate `PublishToSubject` function for publishing directly to a NATS subject.

Reply:
in this context `def _publish()` means publish to stream, so I suggest to keep 

```
response = self.stub.Publish(publish_request)
```

Reply:
https://github.com/liftbridge-io/liftbridge-api/blob/v1.0.0/api.proto#L76
It looks like `PublishRequest` uses a stream name to find the stream to publish for, but existing functionality uses stream subject to publish the message (https://github.com/dgzlopes/python-liftbridge/blob/master/python_liftbridge/message.py#L15). So, we need to change it to `PublishToSubject` to keep this functionality intact.

Reply:
In that case, instead of changing to `PublishToSubject` as of now , how's about changing the `_create_publish_request` method to use `Publish` ? What do you think ?

Reply:
The problem is that the message class interface does not have `stream` (stream name) attribute, meaning we should change from subject attribute to stream name in the `Message` class, in order to continue to use `Publish`?

Reply:
So, all the examples should be changed, etc. to use the stream name instead of the subject?

Reply:
I believe this python API was written pre 1.0 in which `publish` was split into `Publish` (to _stream_) and `PublishToSubject` (to NATS _subject_).

Reply:
@tylertreat @LaPetiteSouris Based on our discussion, I created a new method `publish_to_subject`, that would allow us to publish to the NATS subject. The `publish` will now use the new attribute `stream` in `Message` to publish to the stream with the name. I updated examples in `README.md` as well as in `examples/lift-pub.py`.